### ISSUE TITLE
Isolate Wallet object

### ISSUE THREAD

**Issue by SlnPons** [massalabs/thyra#258]
**Context**
Today, the wallet function is fully integrated into Thyra. The objective of this ticket is to isolate this object and all its functions outside Thyra as a stand alone service that communicates and so allow Thyra users to interact with their wallet as before.
DEFINITION:
An object will be defined as a set of attributes and methods associated to these attributes. 
* An attribute is defined as a property of an object (ex. the number of doors of a car is an attribute of a car model)
* A method is defined as a standard way to achieve a specific functionality (ex. listening to music is a feature of a car that can be activated by using the method “turning on the radio”.)

**Needs**
**_As a user, I want to use Thyra to upload a website and / or check my wallet balance as usual._**
* As a dev, I want the wallet functions and service to exist outside Thyra but still works
**_As a PM, I want to see the sequence diagram that represents the comm between the wallet object and Thyra once this issue is done._**

**How to** 
- [ ]  Externalise the wallet and following functions:
** Create
** Delete
** Load
** Sign 
- **_[ ] Remane the endpoints for them do be more logical and easy to reminder. Check with your PM to validate them_**
- **_[ ] Create sequence diagram and share it in the issue_**

**Technical details**

**Comment by SlnPons** [comment]
@gregLibert could you use this ticket to add the API on the wallet plug-in side please? 
Like this it will enable the team to start working on the plug-in manager side. And others to start improving the wallet code.

**Comment by SlnPons** [comment]
Issue addressed by Gregory on this repo: https://github.com/massalabs/thyra-plugin-massa-core/issues 
The highlighted line in the issue description haven't been addressed and will be added into another ticket.
